Warning for invalid name (#9515)
mendix/docs: content/en/docs/apidocs-mxsdk/apidocs/studio-pro-10/extensibility-api/web/get-started.md

    You will see a new `Extensions` item in the top menu.

{{% alert color="warning" %}}

> Please note that the name `myextension` has no invalid characters. Extension names should only contain digits, letters, dashes or underscores. An invalid name will cause an error to show and the extension will not be loaded.

Reviewer: Not sure I understand this. Do you mean something like:
```suggestion
Extension names used in place of `myextension` must only contain digits, letters, dashes, and underscores. Extensions with an will not be loaded, and will display an error.
```

I assume that valid names can also not start with a dash?

Author: they can start with a dash, nothing in the code to stop that

Reviewer: @ester-nl 
OK - lots of names of things in programming languages can only start with an underscore or a letter. But I guess this is an exception.

Does my suggestion look good?
